plugins/check_http.c

```diff
@@ -1748,6 +1755,10 @@ print_help (void)
   printf ("    %s\n", _("1.2 = TLSv1.2). With a '+' suffix, newer versions are also accepted."));
   printf (" %s\n", "--sni");
   printf ("    %s\n", _("Enable SSL/TLS hostname extension support (SNI)"));
+#if OPENSSL_VERSION_NUMBER >= 0x10002000L
```

Comment:
Why this exact `OPENSSL_VERSION_NUMBER`? Please add a comment at least which explains why this matters.